operations/app/src/modules/metabase/main.tf

```diff
@@ -29,7 +25,7 @@ resource "azurerm_app_service" "metabase" {
   }
 
   app_settings = {
-    "MB_DB_CONNECTION_URI" = var.postgres_url
+    "MB_DB_CONNECTION_URI" = "postgresql://${data.azurerm_postgresql_server.postgres_server.name}.postgres.database.azure.com:5432/metabase?user=${data.azurerm_key_vault_secret.postgres_user.value}@${data.azurerm_postgresql_server.postgres_server.name}&password=${data.azurerm_key_vault_secret.postgres_pass.value}&sslmode=require&ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory"
```

Comment:
This is more a question rather than a request for change:
This uses the following connection Uri scheme:
`postgresql://HOST:PORT/DBNAME?user=<USER>@<HOST>&password=PASSWORD&etc....`

Looking at https://www.postgresql.org/docs/11/libpq-connect.html#id-1.7.3.8.3.6 , the URI scheme suggest using
`postgresql://USER:PASSWORD@HOST:PORT/DBNAME?etc....`

Reply:
Good question. I'm not sure. Metabase was setup by a prior engineer, so I'm not aware of the reasoning.

For this PR, I'm keeping configuration 100% the same so Terraform shows no changes between the old and new configuration.

Reply:
agreed